config/default.yml

```diff
@@ -4055,6 +4055,15 @@ Style/EnvHome:
   Safe: false
   VersionAdded: '1.29'
 
+Style/EscapedQuotes:
+  Description: 'Avoid escaping quotation marks within a string.'
```

Comment:
I think the description here is a bit misleading, given the cop is more about choosing a consistent style when it comes to dealing with quotation marks in strings.